data-structures-csharp/data-structures-csharp/data-structures-csharp.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>CSharp.DataStructures</id>
+    <version>1.2.0.0</version>
+    <title>CSharp.DataStructures</title>
+    <authors>Fabio Costa, Riyad Parvez</authors>
```

Comment:
Can you please remove those lines? Other people have also contributed to this project, it won't be fair to them. I will add a CONTRIBUTORS file at some point.